_i18n/en/resources/moneropedia/atomic-units.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: moneropedia
+entry: "Atomic Units"
+terms: ["atomic-units", "atomic-unit"]
+summary: "Atomic Units refer to the smallest fraction of 1 XMR according to the monerod implementation."
+---
+
+### The Basics
+
+{{ page.summary | capitalize }}.
+This fraction is actually of 10^-12 XMR (0.000000000001 XMR), which equals to 1 @piconero.
```

Comment:
This line is a bit confusing to read too. How about:

>One atomic unit is currently 1e-12 XMR (0.000000000001 XMR, or one @piconero).

Reply:
Sure